src/main/java/seedu/address/logic/commands/ExpenseCommand.java

```diff
@@ -106,47 +150,14 @@ public CommandResult execute(Model model) throws CommandException {
             }
         }
 
-        if (persons.size() > 1) {
-            List<Person> personList = new ArrayList<>();
-            personList.add(payingPerson);
```

Comment:
Any particular reason for not including the payingPerson in the personList? Previously there were warnings if the paying person is duplicated in the list of other involved people, which I thought was good to have just so that users are aware.

Reply:
It's checked for in line 109

Reply:
But currently if you do `expense p/p1 e/10 p/p1` there is actually no warning shown cause the warnings are determined by this code segment and `personList` now doesn't contain the paying person ID
`if (personList.contains(personPriKey)) {
    warningMessage.append(String.format(WARNING_DUPLICATE_PERSON, person.getName()));
    continue;
}`